arboretum/ibm_cloud/README.md

```diff
@@ -18,10 +18,62 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster List
+
+* Class: [ClusterListFetcher][fetch-cluster-list]
+* Purpose: Write the list of IBM Cloud clusters to the evidence locker.
+* Behavior: Log in to IBM Cloud using `ibmcloud login` command, and save the result of `ibmcloud cs cluster ls` command.
```

Comment:
This really isn't the behavior content we're looking for.  You shouldn't explicitly state the command that's getting executed because we'll have update this README anytime any sort of change to the command happens.  We may also in the future move away from the command and towards using the API if/when it stabilizes but the behavior of the fetcher would remain the same.  I think something that communicates the behavior in a more broad (generic) way would be better.

Reply:
I've changed the description into more abstract expression.
https://github.com/ComplianceAsCode/auditree-arboretum/commit/89f20fdadcd3382457262f8d93911bcd5bcbd01c#diff-fa9c900ac18eaea9441dab042c0ad0c9

> Behavior: Log in to IBM Cloud and save the list of clusters bound with specified account.